test/metabase/driver/common/parameters/dates_test.clj

```diff
@@ -11,29 +11,54 @@
             [:field "field" {:base-type :type/DateTime, :temporal-unit :day}]
             "2019-04-01"
             "2019-04-30"]
-           (dates/date-string->filter "2019-04" [:field "field" {:base-type :type/DateTime}])))
```

Comment:
the existing parenthesis was wrong (everything went under `testing "year and month"`) thus the formatting shift